nway/nway_matching.py

```diff
@@ -133,13 +134,29 @@ def make_masks_from_dicts(self):
             path to json file containing nway matching input
 
         """
+        avg_proj_key = "ophys_average_intensity_projection_image"
         self.experiments = []
+        sizes = dict()
         for exp in self.args['experiment_containers']['ophys_experiments']:
             self.experiments.append(
                     utils.create_nice_mask(
                         exp,
                         self.args['output_directory']))
 
+            with PIL.Image.open(exp[avg_proj_key]) as im:
```

Comment:
I wonder if it makes sense to have this size checking logic in its own separate function? Would make things easier to test.

Reply:
that's a good point

Reply:
done